bootstrap/components/topology/storm-topology-component.json

```diff
@@ -55,6 +55,40 @@
             "defaultValue": "hdfs://localhost:9000/tmp/hbase"
           }
         ]
+      },
+      {
```

Comment:
Is this always going to be shown or only if streamline is running in kerberos mode ? Is it optional in non-secure mode ?